public/app/plugins/datasource/cloud-monitoring/components/AliasBy.tsx

```diff
@@ -24,8 +22,8 @@ export const AliasBy: FunctionComponent<Props> = ({ refId, value = '', onChange
   };
 
   return (
-    <QueryEditorRow label="Alias by" htmlFor={`${refId}-alias-by`}>
-      <Input id={`${refId}-alias-by`} width={INPUT_WIDTH} value={alias} onChange={onChange} />
-    </QueryEditorRow>
+    <EditorField label="Alias by">
+      <Input id={`${refId}-alias-by`} width={SELECT_WIDTH} value={alias} onChange={onChange} />
```

Comment:
We can actually remove this entirely since if we don't provide a prop to `Input` the `width` is set to `100%` which means the parent's `min-width` can control it.